src/base-lib/src/constant/constant.ts

```diff
@@ -362,6 +362,12 @@ export const OUTPUT_REPORTING_QUICKSIGHT_DATA_SOURCE_ARN =
 export const OUTPUT_REPORTING_QUICKSIGHT_DASHBOARDS =
   'Dashboards';
 
+export const OUTPUT_REPORTING_QUICKSIGHT_REDSHIFT_SERVERLESS_DATA_API_ROLE_ARN =
```

Comment:
rename it to `OUTPUT_REPORTING_QUICKSIGHT_REDSHIFT_DATA_API_ROEL_ARN` in case we use it for exploratory queries